test/RadworksGovernor.t.sol

```diff
@@ -7,6 +7,21 @@ import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
 import {IGovernorAlpha} from "src/interfaces/IGovernorAlpha.sol";
 import {RadworksGovernorTest} from "test/helpers/RadworksGovernorTest.sol";
 import {ProposalTest} from "test/helpers/ProposalTest.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+
```

Comment:
help me understand, did we combine these test contracts so that they'd play better with `scopelint spec`? did we like the domain separation of the different test files?

I wonder, if we had just imported them (`import from "./RadworksDripsGovernance.sol`) could we get the best of both worlds -- domain separation and also `scopelint spec` compatibility?

Reply:
I've tried a few things in this area to no avail.. scopelint doesn't seem to want to follow the imports.